src/mapping/query/function/mod.rs

```diff
@@ -120,6 +122,70 @@ build_signatures! {
     contains => ContainsFn,
     slice => SliceFn,
     tokenize => TokenizeFn,
+    split => SplitFn,
+}
+
+pub(in crate::mapping) struct TypedArgument {
```

Comment:
Maybe `DynamicArgument` is a better name? Since the other arguments (Regex, but others as well) are also "typed", they just don't resolve to a `Value`.